go/chat/utils/utils_test.go

```diff
@@ -1012,3 +1013,24 @@ func TestTLFIsTeamID(t *testing.T) {
 	tlfID = chat1.TLFID(uid.ToBytes())
 	require.False(t, tlfID.IsTeamID())
 }
+
+func TestGetMessageBodyTxIDs(t *testing.T) {
```

Comment:
this is a unit test `GetMessageBodyTxIDs`, and the existing test `TestChatSrvStellarUI` in `server_test.go` already does a roundtrip test, since it calls `Send()`, which adds the txIDs in the header, and also does an `UnboxMessage()`.